reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -39,262 +39,7 @@ import kotlinx.html.*
-          text-overflow: ellipsis;
-          word-wrap: break-word;
-        }
-
-        .ort-report-table td:last-child {
-          border-right: 1px solid rgba(34,36,38,.15);
-        }
-
-        .ort-report-table.ort-violations tr.ort-resolved {
-          background: #fcfff5;
-          color: #2c662d;
-        }
-
-        .ort-report-table.ort-packages tr.ort-error {
-          color: black;
-        }
-
-        .ort-report-table.ort-packages tr.ort-error td:nth-child(5),
-        .ort-report-table.ort-packages tr.ort-error td:nth-child(6) {
-          color: #9f3a38;
-        }
-
-        .ort-report-table li.ort-resolved {
-          color: #2c662d;
-        }
-
-        @media all and (max-width: 1000px) {
-            .ort-report-table th:nth-child(2), .ort-report-table td:nth-child(2) {
-                display:none;
-                width:0;
-                height:0;
-                opacity:0;
-                visibility: collapse;
-            }
-        }
-
-        @media all and (max-width: 900px) {
-            .ort-report-table th:nth-child(3), .ort-report-table td:nth-child(3) {
-                display:none;
-                width:0;
-                height:0;
-                opacity:0;
-                visibility: collapse;
-            }
-        }
-
-        @media all and (max-width: 800px) {
-            .ort-report-table th:nth-child(5),
-            .ort-report-table td:nth-child(5),
-            .ort-report-table th:nth-child(6),
-            .ort-report-table td:nth-child(6) {
-                display:none;
-                width:0;
-                height:0;
-                opacity:0;
-                visibility: collapse;
-            }
-
-            .ort-report-table th:nth-child(4) {
-              border-top-right-radius: .28rem;
-              border-right: 1px solid rgba(34,36,38,.15);
-            }
-
-            .ort-report-table td:nth-child(4) {
-              border-right: 1px solid rgba(34,36,38,.15);
-            }
-
-            .ort-report-table tr:last-child td:nth-child(4) {
-              border-bottom-right-radius: .28rem;
-            }
-        }
-
-        @media all and (max-width: 500px) {
-            .ort-report-table th:nth-child(4),
-            .ort-report-table td:nth-child(4) {
-                display:none;
-                width:0;
-                height:0;
-                opacity:0;
-                visibility: collapse;
-            }
-
-            .ort-report-table th:first-child {
-              border-top-right-radius: .28rem;
-              border-right: 1px solid rgba(34,36,38,.15);
-            }
-
-            .ort-report-table td:first-child {
-              border-right: 1px solid rgba(34,36,38,.15);
-            }
-
-            .ort-report-table tr:last-child td:first-child {
-              border-bottom-right-radius: .28rem;
-            }
-        }
-
-        """.trimIndent()
+    private val css = javaClass.getResource("/static-html.css").readText()
```

Comment:
Nit: Maybe name the file `static-html-reporter.css`?

Reply:
Done.